packages/carbon-graphs/tests/unit/controls/Graph/GraphPanning-spec.js

```diff
@@ -92,6 +96,128 @@ describe('Graph - Panning', () => {
+          ),
+        ).translate[0],
+      ).toBeCloserTo(60);
+      const panData = {
+        key: 'uid_1',
+        values: [
+          {
+            x: '2016-03-03T12:00:00Z',
+            y: 2,
+          },
+          {
+            x: '2016-04-03T12:00:00Z',
+            y: 20,
+          },
+        ],
+        xLabel: 'updated xLabel',
+      };
+      graph.reflow(panData);
+      graph.resize();
+      triggerEvent(window, 'resize', () => {
+        expect(
+          getSVGAnimatedTransformList(
+            fetchElementByClass(styles.axisX).getAttribute(
+              'transform',
+            ),
+          ).translate[0],
+        ).toBeCloserTo(60);
+        done();
+      });
+    });
+    it('should transform y-axis to the appropriate position', (done) => {
+      graph.destroy();
+      const axisObj = utils.deepClone(axisDefault);
+      axisObj.y.label = ' ';
+      graph = new Graph({ ...getAxes(axisObj) });
+      expect(
+        getSVGAnimatedTransformList(
+          fetchElementByClass(styles.axisY).getAttribute(
+            'transform',
+          ),
+        ).translate[0],
+      ).toBeCloserTo(50);
+      const panData = {
+        key: 'uid_1',
+        values: [
+          {
+            x: '2016-03-03T12:00:00Z',
+            y: 2,
+          },
+          {
+            x: '2016-04-03T12:00:00Z',
+            y: 20,
+          },
+        ],
+        yLabel: 'updated yLabel',
+      };
+      graph.reflow(panData);
+      graph.resize();
+      triggerEvent(window, 'resize', () => {
+        expect(
+          getSVGAnimatedTransformList(
+            fetchElementByClass(styles.axisY).getAttribute(
+              'transform',
+            ),
+          ).translate[0],
+        ).toBeCloserTo(68);
+        done();
+      });
+    });
+    it('should transform y2-axis to the appropriate position', (done) => {
+      graph.destroy();
+      const axisObj = utils.deepClone(axisDefault);
+      axisObj.y2 = {
+        show: true,
+        label: ' ',
+        lowerLimit: 11,
+        upperLimit: 25,
+      };
+      graph = new Graph({ ...getAxes(axisObj) });
+      expect(
+        getSVGAnimatedTransformList(
+          fetchElementByClass(styles.axisY2).getAttribute(
+            'transform',
+          ),
+        ).translate[0],
+      ).toBeCloserTo(930);
+      const panData = {
+        key: 'uid_1',
+        values: [
+          {
+            x: '2016-03-03T12:00:00Z',
+            y: 2,
+          },
+          {
+            x: '2016-04-03T12:00:00Z',
+            y: 20,
+          },
+        ],
+        y2Label: 'updated y2Label',
+      };
+      graph.reflow(panData);
+      graph.resize();
+      triggerEvent(window, 'resize', () => {
+        expect(
+          getSVGAnimatedTransformList(
+            fetchElementByClass(styles.axisY2).getAttribute(
+              'transform',
+            ),
+          ).translate[0],
+        ).toBeCloserTo(930);
+        done();
+      });
+    });
```

Comment:
I think we can add three more tests here: 

- check the height of the canvas container when the label is passed for the x-axis during reflow.
- check the position of the legend when label is passed for the x-axis during reflow.
- check the position of the content container when label is passed to the y-axis during reflow.